src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -102,6 +102,12 @@ module sonic-port{
 						pattern "on|off";
 					}
 				}
+
```

Comment:
Kindly copy the relevant config example, or the link to config here, thx.

Reply:
@praveen-li, for example from `config_db.json`

```
"Ethernet8": {
    "index": "2", 
    "lanes": "8,9,10,11", 
    "fec": "rs", 
    "pfc_asym": "off", 
    "mtu": "9100", 
    "alias": "Ethernet8", 
    "admin_status": "up", 
    "autoneg": "0", 
    "speed": "100000",
},
```

Reply:
Please update the revision, also, add a description for the new field.